packages/astro-reactive-form/core/form-group.ts

```diff
@@ -7,7 +7,9 @@ export class FormGroup {
 
 	constructor(controls: FormControlBase[], name: string = '') {
 		this.name = name;
-		this.controls = controls.map((control) => new FormControl(control));
+		this.controls = controls
+			.filter(control => control.type !== 'submit')
```

Comment:
This is good for now... later we may want to create a type that does not allow a submit control in the controls array. 😉